### PR TITLE
kpackagetool5: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/kpackagetool5.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# kpackagetool5
+
+> KPackage Manager: Install, list, remove Plasma packages.
+> More information: https://techbase.kde.org/Development/Tutorials/Plasma5/QML2/GettingStarted#Kpackagetool5.
```
Comment (sbrl) [inline pages/linux/kpackagetool5.md:4]: Small stylistic thing to match the convention on other pages :-)
```suggestion
> More information: <https://techbase.kde.org/Development/Tutorials/Plasma5/QML2/GettingStarted#Kpackagetool5>.
```

File: pages/linux/kpackagetool5.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- List all known package types that can be installed:
+
+`kpackagetool5 --list-types`
+
+- Install the package from a directory:
+
+`kpackagetool5 -t {{package_type}} -i {{path/to/directory}}`
```
Comment (sbrl) [inline pages/linux/kpackagetool5.md:12]: Is there a long form of any of these arguments? In tldr pages we generally prefer long-form options except in specific circumstances (e.g. `ls`).

Comment (navarroaxel) [reply pages/linux/kpackagetool5.md]: Done!

File: pages/linux/kpackagetool5.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`kpackagetool5 --type {{package_type}} --upgrade {{path/to/directory}}`
+
+- List installed plasmoids (--global for all users):
+
+`kpackagetool5 --type Plasma/Applet --list --global`
+
+- Remove a plasmoid by name:
+
+`kpackagetool5 --type Plasma/Applet --remove "{{name}}"`
```
Comment (navarroaxel) [inline pages/linux/kpackagetool5.md:24]: @sbrl Should I use `"` here? because I didn't use in `{{path/to/directory}}`

**Pull request merged**